resources/web/server/simulation_server.py

```diff
@@ -315,6 +315,7 @@ def runWebotsInThread(client):
                                 dockerComposePath = config['dockerConfDir'] + "/docker-compose-theia.yml"
                                 envVarDocker["THEIA_VOLUME"] = volume
                                 envVarDocker["THEIA_PORT"] = port + 500
+                                client.websocket.write_message('theia: enable')
```

Comment:
I would call it "ide" instead of "theia", as the webots.js GUI button is "ideButton", it's not specific to theia.